.gitignore

```diff
@@ -1,2 +1,3 @@
 /node_modules
 /bower_components
+npm-debug.log
```

Comment:
well we still do have a package.json, and someone could hypothetically run npm install in here. I think it's good practice to keep the debug log and the npm/bower folders hidden in case someone ends up using them. (and the gitignore is tiny so whatevs)